component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPConstants.java

```diff
@@ -40,6 +41,11 @@
     public static final String MOBILE_CLAIM = "http://wso2.org/claims/mobile";
     public static final String SAVED_OTP_LIST = "http://wso2.org/claims/otpbackupcodes";
     public static final String USER_SMSOTP_DISABLED_CLAIM_URI = "http://wso2.org/claims/identity/smsotp_disabled";
+    public static final String SMS_OTP_FAIL_ATTEMPTS_CLAIM = "http://wso2.org/claims/identity/failedSmsOtpAttempts";
```

Comment:
can't we use the same failedLoginAttempts claim here?

Reply:
failedLoginAttempts claim gets reset when we complete basic auth. So, one can avoid account lock by completing basic auth after a few failed attempts for SMS OTP.